README.md

```diff
@@ -69,35 +69,38 @@ $ git clone https://github.com/nvim-treesitter/nvim-treesitter.git
 
 ## Adding parsers
 
-Treesitter uses a different _parser_ for every language. It can be quite a pain to install, but fortunately `nvim-treesitter`
-provides two command to tackle this issue:
+Treesitter uses a different _parser_ for every language, which needs to be generated via `tree-sitter-cli` from a `grammar.js` file, then compiled to a `.so` library that needs to be placed in neovim's `runtimepath` (typically under `parser/{lang}.so`). To simplify this, `nvim-treesitter`
```

Comment:
Isn't it getting to complicated when you explain to newbies how to install or generate custom parsers? We could out-source this into a install your own parser section. But the instructions are quite clear so it could be also fine here.

Reply:
That would be an alternative, yes; I thought of putting it in the wiki, but 

1. the instructions are quite short (effectively replacing the not really necessary example output that was there initially) and
2. especially at the moment, you want to make it as easy as possible for people to write and contribute parsers, so having that as visible as possible makes sense.

Later we can replace this with a link to a Wiki page (especially if the instructions get more extensive with queries etc.), but there should be an easy-to-find place that points to this information.